packages/user-ui/src/pages/page.rs

```diff
@@ -19,21 +20,17 @@ use super::controller;
 pub fn MainPage(lang: Language) -> Element {
     let ctrl = controller::Controller::init(lang.clone())?;
 
-    let public_opinions = ctrl.get_public_opinions();
-    let public_opinion_institutions = ctrl.get_public_opinion_institutions();
-    let public_opinion_reviews = ctrl.get_public_opinion_reviews();
+    let deliberations = ctrl.get_public_opinions();
+    let institutions = ctrl.get_institutions();
+    let public_opinion_reviews = ctrl.get_reviews();
```

Comment:
```suggestion
    let deliberations = ctrl.public_opinions()?;
    let institutions = ctrl.institutions()?;
    let public_opinion_reviews = ctrl.reviews()?;
```
DioxusController 를 사용하면 이런식으로 할 수 있습니다.

Reply:
아 넵 수정했습니다!